src/main/webapp/app/index.tsx

```diff
@@ -8,6 +8,74 @@ import setupAxiosInterceptors from './config/axios-interceptor';
 import ErrorBoundary from './shared/error/error-boundary';
 import AppComponent from './app';
 import { loadIcons } from './config/icon-loader';
+import { AppConfig } from './appConfig';
+import * as Sentry from '@sentry/react';
+
+if (AppConfig.serverConfig?.frontend.sentryDsn) {
```

Comment:
After we change the frontend properties to optional, we should also add `?.sentryDsn`